### PR TITLE
Sort by Ratings

### LINKED ISSUE #14: Create Sort By Ratings
- Add two more filters to sort by ratings

### PR DESCRIPTION
**Pull request by ttahm3d**
Added sort by ratings #14

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
All Code looks good

File: src/hooks/useAxios.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//       }
+//     } catch (e) {
+//     } finally {
+//     }
+//   };
+
+//   useEffect(() => {}, []);
+
+//   return { response, error, isLoading };
+// }
```
Comment (rdshinde) [inline src/hooks/useAxios.jsx:23]: Remove this if not needed.

**Review by therajatg — APPROVED** [review]
looks clean and good to merge

**Review by rdshinde — APPROVED** [review]
A small confirmation, is it ok to use `rem` for small values because you have used it for values like `0.25rem`, `0.75rem` If it is not relative, can't we use pixels?

Otherwise everything is fine,

**Pull request merged**